packages/workers-utils/src/config/validation.ts

```diff
@@ -1056,6 +1057,58 @@ const validateTailConsumers: ValidatorFn = (diagnostics, field, value) => {
 	return isValid;
 };
 
+function validateStreamingTailConsumer(
+	diagnostics: Diagnostics,
+	field: string,
+	value: StreamingTailConsumer
+) {
+	if (typeof value !== "object" || value === null) {
+		diagnostics.errors.push(
+			`"${field}" should be an object but got ${JSON.stringify(value)}.`
+		);
+		return false;
+	}
+
+	let isValid = true;
+
+	isValid =
```

Comment:
will need to be fromatted
```suggestion
	let isValid = 
```